chart/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.0
```

Comment:
Could this be swapped to use the tag or release when the workflow is ran?  

Incrementing this on each release is annoying.  `helm package . --version $version` is a bit more straight forward.

Reply:
i'm not sure of the actual CI variable name for getting the tag. $version is a placeholder

Reply:
@zoriya i figured you'd just combine both the Kyoo release and the helm chart release versions together.  Technically the halm chart can be versioned independently.  Just didn't think you'd want to go through the effort.

Reply:
Updated, but I think I need to test it first.